### PR TITLE
Dec Dev Update

### PR DESCRIPTION
**Pull request by whilei**
As always, feedback welcome!

### CODE REVIEW

File: content/blog/devupdate/2017-12-1.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+=== projects
+
+**emerald-wallet** will release their first https://github.com/ethereumproject/emerald-wallet/releases/latest[_Release Candidate_ (RC1)] today, including support for BIP39 mnemonic phrase retrieval, connection with Shapeshift, improved loading and notification UI, a landing page, and a big refactoring for style and linting. Supporting the wallet's backend, **emerald-cli** released https://github.com/ethereumproject/emerald-cli/releases/latest[v0.18.0] also with support for BIP39 via JSON-RPC and a few minor bug fixes, and **emerald-rs (Vault)** released v0.17.0 and https://github.com/ethereumproject/emerald-rs/releases/latest[v0.18.0] with support for BIP39 mnemonics and BIP32 key derivation. Finally, if you're a React dev, DAPP dev, or chocolate enthusiast, check out https://github.com/shanejonas/react-box-web3-todo[@shanejonas's React TODO Truffle Box], which can quickly put you within throwing distance of your next most ambitious DAPP project.
```
Comment (splix) [inline content/blog/devupdate/2017-12-1.adoc]: "will release" doesn't work for update. let's write about what happened, not about what's going to happen

File: content/blog/devupdate/2017-12-1.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+**go-ethereum** got an important improvement with a fix for peer total difficulty
+calculation, which should fix an issue around a "stalling" issue several users have
+reported. We're currently nearing the end of a major refactor/redesign of logging,
+and are aiming for a release this week. Until then, feel free to jump in early with
+the head tracking fix at https://github.com/ethereumproject/go-ethereum/releases/tag/v4.1.2[prerelease v4.1.2] if the stalling issue has been bothering you. In other news, @whilei and @shanejonas made some experiments in log analysis with geth's forthcoming `mlog` API and Elastic's ELK stack. You can find a walkthrough, configs, and some pretty pictures for that at https://github.com/whilei/geth-ELK[whilei/geth-ELK].
```
Comment (splix) [inline content/blog/devupdate/2017-12-1.adoc]: Same here, it didn't happen yet, so not writing

File: content/blog/devupdate/2017-12-1.adoc [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+title: December Dev Update
+published: true
+date: 04.12.2017 09:00
+---
+
+=== community
+
+Several of us from the team and many from the community were able to meet face-to-face for the first time _ever_ in Hong Kong at the _also-first-ever_ and definitely-hopefully-first-_annual_ ETC Summit! A big thank you to the event organizers _and_ to so many wonderful participants from
+near and far who made it such a diverse, energetic, and welcoming space. Please note
+that recordings of the entire (I think?) summit's presentations can be found
```
Comment (gagarin55) [inline content/blog/devupdate/2017-12-1.adoc]: remove "(I think?)" please

File: content/blog/devupdate/2017-12-1.adoc [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+=== projects
+
+**emerald-wallet** will release their first https://github.com/ethereumproject/emerald-wallet/releases/latest[_Release Candidate_ (RC1)] today, including support for BIP39 mnemonic phrase retrieval, connection with Shapeshift, improved loading and notification UI, a landing page, and a big refactoring for style and linting. Supporting the wallet's backend, **emerald-cli** released https://github.com/ethereumproject/emerald-cli/releases/latest[v0.18.0] also with support for BIP39 via JSON-RPC and a few minor bug fixes, and **emerald-rs (Vault)** released v0.17.0 and https://github.com/ethereumproject/emerald-rs/releases/latest[v0.18.0] with support for BIP39 mnemonics and BIP32 key derivation. Finally, if you're a React dev, DAPP dev, or chocolate enthusiast, check out https://github.com/shanejonas/react-box-web3-todo[@shanejonas's React TODO Truffle Box], which can quickly put you within throwing distance of your next most ambitious DAPP project.
```
Comment (gagarin55) [inline content/blog/devupdate/2017-12-1.adoc]: remove "will" and "today" please, just the fact

**Review by splix — APPROVED** [review]

**Pull request merged**